openai/assistant_event_handler.py

```diff
@@ -22,7 +22,7 @@ class EventHandler(AssistantEventHandler):
     def __init__(self, config, params, last_message_id):
         super().__init__()
         self.run_id = None
-        self.response = {'status': 1, 'messages': []}
+        self.response = {'status': 0, 'message': []}
```

Comment:
ideal way is to defne error codes for each case like 
ERR_SUCCESS = 0
ERR_INCOMPLETE_MAX_TOKEN_REACHED=1
ERR_EXPIRED_TIMEOUT=2